src/styles/block/watch.scss

```diff
@@ -0,0 +1,46 @@
+.watch {
+  width: $watchSize;
+  height: $watchSize;
+  border-radius: 50%;
+  background-color: rgb(255, 233, 92);
+  border: 4px solid rgb(0, 0, 0);
+  cursor: pointer;
+
+  &__center {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    width: 30px;
+    height: 30px;
+    border-radius: 50%;
+    transform: translate(-50%, -50%);
+    background-color: black;
+    z-index: 4;
+  }
+
+  &__hand-min {
+    position: absolute;
+    left: 50%;
+    top: 3%;
+    transform: translateX(-50%);
+    border-radius: 40%;
+    height: $handHeightMin;
+    width: 5px;
+    background-color: black;
+    transform-origin: center bottom;
+    animation: moveHand 3600s infinite steps(60);
+  }
+
+  &__hand-sek {
+    position: absolute;
+    left: 50%;
+    top: 25%;
+    transform: translateX(-50%);
```

Comment:
transform: translateX(-50%);  it didn't work for me at all. As soon as I turned on the animation, the arrows moved to the right. So I fixed it this way  
` left:0;
    right:0;
    margin:0 auto;`

Reply:
your solution works, thanks)